### ISSUE TITLE
[Feature Request] In seyes mode, lines instead of grid

### ISSUE THREAD

**Issue by SeeDborg** [OpenBoard-org/OpenBoard#877]
Hi,
I use the seyes mode in OpenBoard 1.7.0

When activated, the "lines" background disappears. I would find it more logical that seyes replaces grid, and we can still use the lines. What do you think about it ?

**Comment by letsfindaway** [comment]
I think the current choice has the background that lines are typically for writing and the grid for math. So it is logical that Seyes replaces the lines. But you might have other arguments. I think in the end a more flexible handling of background ruling would be the best solution. Here in Germany nobody cares about Seyes, but instead we have two types of lines for primary school. A music teacher would like to have Stave lines, we have already seen the request for an isometric grid and so on. 

So my request would be to provide more flexibility.

**Comment by SeeDborg** [comment]
I agree, more flexibility for the background would be the better option.

Until we have that, wouldn't it be possible to simply have to choose between the 4 backgrounds without having to sacrificing one of them ?

**Comment by sebojolais** [comment]
Hi,

> lines are typically for writing and the grid for math

Yes, it was the idea. My experience is, in the other side of Seyes background, I need grid background oftener than lines. But it is my use-case and I imagine there are plenty others.
The other constraint was no change in the GUI.
More flexibility in background management should be great, but we need to change the GUI. It will be an new feature request where more specifications should be welcome.

**Comment by SeeDborg** [comment]
I agree.
Not having to choose and being able to use the 4 backgrounds would be the best of two worlds :)

**Comment by sebojolais** [comment]
I speak about specifications and specifications arrive ! So nice, great job !
I will read that in details and provide a feedback soon.

**Comment by sebojolais** [comment]
I think that this Feature request is covered by  the #888 PR.
